examples/svelte/spreadsheet/src/index.css

```
@@ -0,0 +1,867 @@
+
+.column-resizer:hover,
+.column-resizer-active {
+  background: #217346;
+}
+
+.spreadsheet-cell {
+  --edge-top: 0;
+  --edge-right: 0;
+  --edge-bottom: 0;
+  --edge-left: 0;
+  display: flex;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)